### DISCUSSION TITLE
DataTable 如何在有selection 列的情况下点击某行跳转

### DISCUSSION THREAD

**Discussion by Vector6662** [tusen-ai/naive-ui#6495] [Q&A]
如题
如果使用row-props属性能够实现点击跳转（vue router）操作，**但会影响左侧selection功能**。

<img width="1850" alt="image" src="https://github.com/user-attachments/assets/cc6978ef-01f6-4275-ab84-f351eeb23cf4">

即，点击第一列的selection框此时也会跳转，而不会进行选中。
```ts
const rowProps = (row: Object) => {
  return {
    style: 'cursor: pointer;',
    onClick: () => {
      this.$router.push({
            path: `/flow/${row.ID}` // 使用vue router路由
          })
    }
  }
}
```

请问有更好的实现方式吗？

**Comment by jahnli** [comment]
可以自定义 checkbox

**Reply by Vector6662** [reply]
> 可以自定义 checkbox

感谢回答，请问能够给具体一点的描述吗？谢谢
我认为即使自定义checkbox，如果仍然使用`row-props`，最终还是会影响自定义checkbox功能。
